### DISCUSSION TITLE
can google adsense work in a player like youtube ?

### DISCUSSION THREAD

**Discussion by Mnshawaty** [kslimani/clappr-ima-plugin#54] [Q&A]
can google adsense work in a player like youtube ?
ok i got how to install the player 
but is it possible to show google adsenese ads on the player ?

**Comment by kslimani** [comment]
Yes this plugin can display google adsense tag url (or any VAST tag url video ad)

**Reply by Mnshawaty** [reply]
yes but please how i can put the adsense ads inside it :(
is there VAST tag for adsense ?

**Reply by kslimani** [reply]
This is a little out of the scope of this plugin, but yes you can generate VAST tag with adsense. Check [the docs](https://support.google.com/adsense/answer/6054303?hl=en).
